serverless.yml

```diff
@@ -14,19 +14,24 @@ provider:
             - "xray:PutTelemetryRecords"
```

Comment:
1. dont you need to allow the s3 put get objects here in the iam section, same for the sqs on the recieve, delete etc

Reply:
I haven't made any changes so it's whatever was there before. Though agreed, the serverless setup needs a proper review.

This PR just lets us utilise the debug features in vscode.